framework/test/gtest/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#-----------------------------------------------------------------------------
+# Link to GTest Libraries
+#-----------------------------------------------------------------------------
+
+if(GTEST_INCLUDE_DIRS)
+  include_directories(${GTEST_INCLUDE_DIRS})
+  add_executable(UnitTests main.cpp)
+  target_link_libraries(UnitTests ${GTEST_BOTH_LIBRARIES})
+  target_link_libraries(UnitTests ${Framework_TARGET})
+
+  # This copies the Google Test libraries into the same folder as the
+  # executable so that they can be seen at runtime on Windows.
+  # Mac and Linux use RPATHs and do not need to do this.
+  if (WIN32)
+    foreach( input_file ${GTEST_BOTH_LIBRARIES})
+      string(REPLACE ".lib" ".dll" input_dll_file ${input_file})
```

Comment:
The .lib files aren't getting copied. `GTEST_BOTH_LIBRARIES` is equal to `<path_to_lib>/gtest_main.lib` and `<path_to_lib>/gtest.lib`. Inside the loop, the .lib extension is replaced with .dll. This ensures both the .dll's are copied to the UnitTests executable directory.